types/smpte-timecode/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+       // TODO
+        "no-any-union": false
```

Comment:
Can you add a comment for why this is disabled? I'm not sure I see why this is necessary.

Reply:
I removed unnecessary rule and commit. Thanks for comment!